src/node-saml/saml.ts

```diff
@@ -1075,11 +1076,17 @@ class SAML {
           if (confirmData && confirmData.$) {
             const subjectNotBefore = confirmData.$.NotBefore;
             const subjectNotOnOrAfter = confirmData.$.NotOnOrAfter;
+            const maxTimeLimitMs = this.processMaxAgeAssertionTime(
+              this.options.maxAssertionAgeMs,
+              subjectNotOnOrAfter,
+              assertion.$.issueInstant
```

Comment:
@cjbarth thanks to you catching the `NaN` issue, I found this capitalization bug which had previously gone undetected! 🙏